src/Http/Http.Results/test/ResultsOfTTests.Generated.cs

```diff
@@ -84,7 +84,7 @@ public void ResultsOfTResult1TResult2_Throws_InvalidOperationException_WhenResul
         // Arrange
         Results<ChecksumResult1, NoContent> MyApi()
         {
-            return new ChecksumResult1(1);
+            return (ChecksumResult1)null;
```

Comment:
Am I reading correctly that the test was looking for an argument null exception but the argument wasn't null?

Reply:
Yes, although it throws an `InvalidOperationException` because the null value here is not an argument but the return type of a method we've indicated cannot be null.